doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -401,7 +401,13 @@ Binary libraries
 Bluetooth libraries and services
 --------------------------------
 
-|no_changes_yet_note|
+* :ref:`bt_fast_pair_readme` library:
+
+  * Updated:
+
+    * Covered new integration steps in the :ref:`ug_bt_fast_pair` guide.
+      Reorganized extension-specific content into dedicated subsections.
```

Comment:
```suggestion
    * Integration steps in the :ref:`ug_bt_fast_pair` guide.
      Reorganized extension-specific content into dedicated subsections.
```
This could be moved to documentation section as this is not specific to library.